kvdb-web/Cargo.toml

```
@@ -0,0 +1,25 @@
+[package]
+name = "kvdb-web"
+version = "0.0.1"
```

Should be at least 0.1.0, so that we can publish patches.

```suggestion
version = "0.1.0"
```